packages/slate/src/models/node.js

```diff
@@ -1622,8 +1623,33 @@ class Node {
    */
 
   getTextsAtRange(range) {
-    const array = this.getTextsAtRangeAsArray(range)
-    return new List(array)
+    range = range.normalize(this)
+    if (range.isUnset) return []
```

Comment:
Seems like this should be `return new List()` instead?

Reply:
Fixed~